### DISCUSSION TITLE
Help enabling and configure Library (Pull 3124)

### DISCUSSION THREAD

**Discussion by wes1993** [immich-app/immich#3195] [Q&A] [answered]
Hello to everyone,
i hope someone could help me, i want to enable the option do add the Local Library in my Instance, i have seen i shold do a POST request to /api/library but when i try i have a 404 error, probably i got something wrong :-( my Server Version is v1.66.1.

I have used restman for enabling Library:

![image](https://github.com/immich-app/immich/assets/52070589/54ee3d0b-a2da-45e6-a4d9-4f3069253d34)

I hope in your help
Best regards
Stefano

**Comment by bo0tzz** [comment, marked as answer]
This feature hasn't been merged and released yet.

**Comment by wes1993** [comment]
Thanks a lot @bo0tzz for your reply :-D

Can i use the beta to test this functionality?

**Reply by bo0tzz** [reply]
No, there is no beta and the feature is not ready yet. You can check out the branch and start the development environment if you have a need to, but there's probably no reason for that ;)

**Reply by etnoy** [reply]
No, this is currently pre alpha and you shouldn't try that yet

**Reply by wes1993** [reply]
Thanks @etnoy, and thanks to everyone for the wonderful work :-D
P.S. it's a bit out of topic but can i help to translate the webserver GUI in Italian?

**Comment by wes1993** [comment]
Ok thanks a lot @bo0tzz for your help :-D
Ok now i need to study how to configure OAUTH and Reverse Proxy :-D

Best regards
Stefano
